### DISCUSSION TITLE
想知道imtoken，有在线收款的api吗？

### DISCUSSION THREAD

**Discussion by bigcat777777** [consenlabs/webview#65] [General]
我有一个棋牌游戏的产品，想直接在内部做在线收款基于trc和erc的usdt，想知道用户每次转账成功后，我的服务器收到通知，进行自动发货

**Comment by xwartz** [comment]
imToken 没有提供这样的 API，你可以监控自己地址的收款

**Reply by bigcat777777** [reply]
那我监控自己的收款地址，有对应的api接口吗？

**Reply by xwartz** [reply]
可能理解错了，imToken 没有提供这样的 API，需要你自己实现。
